### PR TITLE
Export account_id as well. to fix not able to edit imported records

### PR DESCRIPTION
**Pull request by JanaHoch**
Currently you can't edit records which were imported from CSV file. With this changes We will be exporting account_id as well so when importing we can put all records in their own accounts. 

Limitation: If you have more than 1 account then you need to create those accounts before importing CSV file.

This will fix this https://github.com/xorum-io/open_money_tracker/issues/6 issue.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -78,12 +78,12 @@
...
     <string name="import_help">To import records into application:\n
     1. Create CSV file in format - \n
-    <b>time;title;category;price;currency.</b>\n
+    <b>time;account_id;title;category;price;currency.</b>\n
     2. Copy its content to text field.\n
     3. Press import button. In few seconds you will see how much records has been imported.\n
     \n
-    <b>Note:</b> time - timestamp value in milliseconds; currency - 3 symbols code of currency.\n
-    <b>Example:</b> 1466948795712;metro;transport;-20.0;UAH</string>
+    <b>Note:</b> time - timestamp value in milliseconds; Default account_id is 1 and for every new account it increases by 1; Make sure all the accounts already exists; currency - 3 symbols code of currency.\n
```
Comment (yev-kanivets) [inline app/src/main/res/values/strings.xml:85]: Nit: `exists`, not `existS`. Can you also add translations in other languages? (Google Translate will do).

**Review by yev-kanivets — COMMENTED** [review]
Hey @JanaHoch, thanks a lot for your contribution 🙏 It looks and works well, just a nitpick regarding wording and translation.

**Comment by JanaHoch** [comment]
Hey @yev-kanivets  I added translations as well. check this 06445a8fe7fbb804ebcb6aa22acffc118034aac4

**Review by yev-kanivets — APPROVED** [review]
Thanks!

**Comment by JanaHoch** [comment]
@yev-kanivets I created a new feature but those commits are also showing up under this PR. And if I try to create a new PR it is also taking these previous commits as well. I checked stack overflow but I'm new to git.

new feature commits: aa8d19e64cff9c3fbd80b52c1608d2df33b87d64 aa8d19e64cff9c3fbd80b52c1608d2df33b87d64 aa8d19e64cff9c3fbd80b52c1608d2df33b87d64 aa8d19e64cff9c3fbd80b52c1608d2df33b87d64 aa8d19e64cff9c3fbd80b52c1608d2df33b87d64 aa8d19e64cff9c3fbd80b52c1608d2df33b87d64

**Pull request merged**